node/pkg/reporter/utils.go

```diff
@@ -86,41 +86,35 @@ func ProofsToMap(proofs []Proof) map[int32][]byte {
 	return m
 }
 
-func MakeContractArgsWithProofs(aggregates []GlobalAggregate, submissionPairs map[int32]SubmissionPair, proofMap map[int32][]byte) ([][32]byte, []*big.Int, []*big.Int, [][]byte, error) {
+func MakeContractArgsWithProofs(aggregates []GlobalAggregate, submissionPairs map[int32]SubmissionPair) ([][32]byte, []*big.Int, []*big.Int, error) {
```

Comment:
I think the function name should change now since it is not returning proofs?

Reply:
I think this function might not be required or could be much simplified. 
please check logic in `./node/script/test_dal_consumer/main.go`

https://github.com/Bisonai/orakl/blob/50d01e323f9089c08930b282fec28c9c833c1f1c/node/script/test_dal_consumer/main.go#L41-L65

Reply:
yes, i think this can be removed. I'll double check

Reply:
I'll make another clean up PR, where most of the code from `utils` will be gone